server/ap_server/models.py

```diff
@@ -260,6 +260,16 @@ class Thread(BaseModel):
     )
```

Comment:
Are we missing an update to "ThreadPatch" to let us directly patch the current thread state? Or is that deliberately omitted?